.github/workflows/build.yml

```diff
@@ -3,13 +3,51 @@ name: Build CHT Conf and test against node versions
 on: [push, pull_request]
 
 jobs:
-  build:
+  ubuntu:
     name: Build for Node version ${{ matrix.node-version }}
     runs-on: ubuntu-18.04
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x]
+        node-version: [8.x, 10.x, 12.x, 14.x, 16.x]
```

Comment:
It looks like you can specify the `runs-on` property in a matrix which I think would remove this duplicated code. Check this example out: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix